### PR TITLE
added diagrams

### LINKED ISSUE #7: Add Architecture Diagram
### Task

- [x] Create and design Architecture Diagram
- [x] Make any changes following feedback
- [x] Get approval
- [x] Add Architecture Diagram

### User Story

As a developer, I want to be able to see the diagrams so I can understand the code logic

### LINKED ISSUE #11: Add Entity Relationship Diagram
### Task

- [x] Create and design ERD
- [x] Make any changes following feedback
- [x] Get approval
- [x] Add ERD to repo

### User Story

As a developer, I want to view the ERD so I can understand how the data is stored in the database

### PR DESCRIPTION
**Pull request by A-Faris**
Added the ERD and Architecture diagram
closes #7, closes #11

### CODE REVIEW

File: diagrams/Architecture_Diagram.png
Comment (FarihaChoudhury) [inline diagrams/Architecture_Diagram.png:1]: I think the boxes seem inconsistent, there is no box for the SNS alert or the dashboard but there is for everything else

File: diagrams/ERD.png
Comment (FarihaChoudhury) [inline diagrams/ERD.png:1]: I think it would be easier to read if the tables were spaced out a little bit more.

Comment (A-Faris) [reply diagrams/ERD.png:1]: Done

File: diagrams/Architecture_Diagram.png
Comment (A-Faris) [reply diagrams/Architecture_Diagram.png:1]: Done

**Review by FarihaChoudhury — APPROVED** [review]
Looks beautiful

**Pull request merged**